tools/openblas_support.py

```diff
@@ -156,6 +158,10 @@ def setup_openblas(plat=get_plat(), ilp64=get_ilp64()):
     if not plat:
         raise ValueError('unknown platform')
     typ = download_openblas(tmp, plat, ilp64)
+    openblas_version = "HEAD" if nightly else OPENBLAS_LONG
+    typ = download_openblas(
+        tmp, plat, ilp64, openblas_version=openblas_version
+    )
```

Comment:
I think this logic belongs in `download_openblas()`, not here

Reply:
I might be a bit fuzzy brained, but I think this is fine? Just I had duplicated the `typ` assignment, which I've now removed.

Reply:
My review relates to the construction of openblas_version here, rather than inside `download_openblas`. To me it makes more sense to do all the URL construction and logic in one place, and not have a piece here and a piece inside `download_openblas`. So I would prefer the signature of `download_openblas` get a `nightly=False` kwarg instead of an `openblas_version` kwarg.

Reply:
Ah I see makes sense, updated.